ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -0,0 +1,646 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class AvatarDemoController: UITableViewController {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(style: .grouped)
+
+        initDemoAvatars()
+    }
+
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(TableViewCell.self, forCellReuseIdentifier: TableViewCell.identifier)
+        tableView.register(BooleanCell.self, forCellReuseIdentifier: BooleanCell.identifier)
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return AvatarDemoSections.allCases.count
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return AvatarDemoSections.allCases[section].rows.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let section = AvatarDemoSections.allCases[indexPath.section]
```

Comment:
I think organizing the demo by style then size seemed a bit cleaner, what do you think? Was it so that each table cell would have consistent height?

Reply:
I agree it does have the benefit of uniform cell height in each section. It also shows all the different looks that the Avatar can have in a single section, which seems to be more straightforward.